Assignment/solution_student/index.html

```diff
@@ -2,32 +2,78 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>FEWD HTML Template</title>
-    <meta name="description" content="An HTML Starter Template">
-    
+    <title>Relaxr</title>
+    <meta name="description" content="A Relaxr Landing page">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,600italic,400italic,800,800italic' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
     <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
-
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
+
   <body>
 
     <!-- YOUR MARKUP GOES HERE! -->
-    <div class="container">
-    
       <header>
-        <h1>FEWD HTML Template</h1>
+
+        <h1>Relaxr</h1>
+
+      <div class= "center-text">
+
+        <h2>Get piece of mind with a single tap</h2>
+
+        <a id="getitnow" href="#">Get It Now</a>
+
+      </div>
+
       </header>
 
-      <main>
-        <p>Our HTML starter template!</p>
+      <div class= "benefits">
+
+        <h3>Benefits</h3>
+
+        <p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+
+        <ul>
+          <li>Schedule meetings for you</li>
+          <li>Excel automation to complete your work for you</li>
+          <li>Responds to emails on your behalf</li>
+          <li>Does all your work for you with our revolutionary AI technology</li>
+        </ul>
+
+      </div>
+
       </main>
 
+      <div class= "center-text">
```

Comment:
Whoops! Looks like you might have just forgotten to remove this closing `</main>` tag — no biggie! :)